### ISSUE TITLE
v3 error: package [PRJ NAME]/helpers is not in GOROOT

### ISSUE THREAD

**Issue by frederikhors** [web-ridge/gqlgen-sqlboiler#42]
I'm getting this error now:

```
9:43PM DBG write GraphQL schema to disk bytes=7869 file=schema.graphql
9:43PM DBG [convert] get boiler models
9:43PM DBG [convert] get extra's from schema
9:43PM DBG [convert] get model with information
9:43PM WRN no models found in graphql so skipping generation
9:43PM DBG [resolver] get boiler models
9:43PM DBG [resolver] get models with information
9:43PM DBG [resolver] generate file
error while trying generate resolver and converts
validation failed: packages.Load: resolver.go:6:2: package api_gqlgen_sqlboiler/helpers is not in GOROOT (c:\go\src\api_gqlgen_sqlboiler\helpers)
C:\api-gqlgen-sqlboiler\resolver.go:6:4: could not import api_gqlgen_sqlboiler/helpers (invalid package name: "")
C:\api-gqlgen-sqlboiler\resolver.go:19:10: undeclared name: queryResolver
C:\api-gqlgen-sqlboiler\resolver.go:19:77: Node not declared by package graphql_models
C:\api-gqlgen-sqlboiler\resolver.go:6:2: "api_gqlgen_sqlboiler/helpers" imported but not used

exit status 3
```

I'm using this go.mod file:

```
module api_gqlgen_sqlboiler

go 1.15

require(...)
```

**Comment by frederikhors** [comment]
I need to execute `go run convert_plugin.go` two times to make it work.

I think we should close this when fixed.

**Comment by RichardLindhout** [comment]
I'm not sure what triggers this issue. Are you generating the resolvers before the converts?

**Comment by frederikhors** [comment]
> Are you generating the resolvers before the converts?

Nope. I'm following the Readme steps.

**Comment by RichardLindhout** [comment]
I think ```cfg, err := config.LoadConfigFromDefaultLocations()``` needs to be after ```err := gbgen.SchemaWrite(gbgen.SchemaConfig{```

I also create the folder if not exist in new version which will be released today 3.1.1

**Comment by RichardLindhout** [comment]
Let me know if this still happens after you change order and I will will re-open the issue :)
